src/server/session.ts

```diff
@@ -1454,7 +1466,7 @@ namespace ts.server {
         private createCheckList(fileNames: string[], defaultProject?: Project): PendingErrorCheck[] {
             return mapDefined<string, PendingErrorCheck>(fileNames, uncheckedFileName => {
                 const fileName = toNormalizedPath(uncheckedFileName);
-                const project = defaultProject || this.projectService.getDefaultProjectForFile(fileName, /*ensureProject*/ false);

```

Comment:
Whats this change for ?

Reply:
Fixes some code that was fishy under `--strictNullChecks` -- if the flag was passed this function returned a non-nullable type.

Reply:
No! Earlier change was correct because we do not want to update graph to get project at this point. We would do that after checklist is completed.